### ISSUE TITLE
False negative on consts used in array sizes in function parameters

### ISSUE THREAD

**Issue by 200sc** [opennota/check#38]
I have the following code.

```Go
const (
	playerAmmoSize = class.LASTAMMO - (class.FIRSTAMMO + 1)
)

func RenderAmmo(f *render.Font, playerAmmos [playerAmmoSize]int, x, y float64) ([]render.Renderable, []*ui.Uibox) {
    ...
```

Through gometalinter (which hopefully has the most recent version of varcheck),
this returns this error:

`tools\firearms\firearm.go:467:2:warning: unused global variable playerAmmoSize (varcheck)`

I can replicate this with this simpler example:

```Go
const (
	varchecktest = 10
)

func VarCheckTest(t [varchecktest]int) {
}
```
`tools\firearms\firearm.go:467:2:warning: unused global variable varchecktest (varcheck)`

**Comment by opennota** [comment]
Thanks for reporting!
